src/Microsoft.Health.Fhir.SqlServer/Features/Search/Expressions/Visitors/MissingSearchParamVisitor.cs

```diff
@@ -28,7 +28,7 @@ public override Expression VisitSqlRoot(SqlRootExpression expression, object con
             {
                 TableExpression tableExpression = expression.TableExpressions[i];
 
-                if (tableExpression.NormalizedPredicate.AcceptVisitor(Scout.Instance, null))
+                if (tableExpression.Kind != TableExpressionKind.Sort && tableExpression.NormalizedPredicate.AcceptVisitor(Scout.Instance, null))
```

Comment:
please add a comment here to explain this if statement, why sort is a special case here